apps/wpcom-block-editor/src/wpcom/features/tracking/track-record-event.js

```diff
@@ -5,13 +5,25 @@
 import { isObjectLike, isUndefined, omit } from 'lodash';
 import debug from 'debug';
 
+/**
+ * Internal dependencies
+ */
+import { isE2ETest, isSandboxed } from '../../../utils';
+
 const tracksDebug = debug( 'wpcom-block-editor:analytics:tracks' );
+const e2ETracksDebug = debug( 'wpcom-block-editor:e2e' );
 
 // In case Tracks hasn't loaded.
 if ( typeof window !== 'undefined' ) {
 	window._tkq = window._tkq || [];
 }
 
+const E2E_STACK_SIZE = 20;
+if ( isSandboxed() || isE2ETest() ) {
```

Comment:
I think we should remove the `isSandboxed()` here. I left it here just for testing purposes. However, it won't work either since it tries to read a global variable that hasn't been added to the source code (dotcom).

```suggestion
if ( isE2ETest() ) {
```

Reply:
Agreed.